examples/pytorch_simple.py

```diff
@@ -98,41 +98,50 @@ def objective(trial):
             if batch_idx * BATCHSIZE >= N_TRAIN_EXAMPLES:
                 break
 
-            data, target = data.view(-1, 28 * 28).to(DEVICE), target.to(DEVICE)
+            data, target = data.view(data.size(0), -1).to(DEVICE), target.to(DEVICE)
 
-            # Zeroing out gradient buffers.
             optimizer.zero_grad()
-            # Performing a forward pass.
             output = model(data)
-            # Computing negative Log Likelihood loss.
             loss = F.nll_loss(output, target)
-            # Performing a backward pass.
             loss.backward()
-            # Updating the weights.
             optimizer.step()
 
-    # Validation of the model.
-    model.eval()
-    correct = 0
-    with torch.no_grad():
-        for batch_idx, (data, target) in enumerate(valid_loader):
-            # Limiting validation data.
-            if batch_idx * BATCHSIZE >= N_VALID_EXAMPLES:
-                break
-            data, target = data.view(-1, 28 * 28).to(DEVICE), target.to(DEVICE)
-            output = model(data)
-            pred = output.argmax(dim=1, keepdim=True)  # Get the index of the max log-probability.
-            correct += pred.eq(target.view_as(pred)).sum().item()
+        # Validation of the model.
+        model.eval()
+        correct = 0
+        with torch.no_grad():
+            for batch_idx, (data, target) in enumerate(valid_loader):
+                # Limiting validation data.
+                if batch_idx * BATCHSIZE >= N_VALID_EXAMPLES:
+                    break
+                data, target = data.view(data.size(0), -1).to(DEVICE), target.to(DEVICE)
+                output = model(data)
+                # get the index of the max log-probability
```

Comment:
A nit, for style and consistency.
```suggestion
                # Get the index of the max log-probability.
```